ipaserver/install/httpinstance.py

```diff
@@ -441,7 +446,8 @@ def __import_ca_certs(self):
     def __publish_ca_cert(self):
         ca_db = certs.CertDB(self.realm, nssdir=paths.HTTPD_ALIAS_DIR,
                              subject_base=self.subject_base)
-        ca_db.publish_ca_cert(paths.CA_CRT)
+        ca_nickname = ca_db.find_root_cert(self.cert_nickname)[-1]
+        ca_db.export_pem_cert(ca_nickname, paths.CA_CRT)
```

Comment:
Ah, right, I did not realize this will be set to the correct value during the initialization from pkcs12.